Misc/NEWS.d/next/Library/2020-04-15-17-21-48.bpo-40287.-mkEJH.rst

```diff
@@ -0,0 +1 @@
+Fixed ``SpooledTemporaryFile.seek()`` did not return the position.
```

Comment:
```suggestion
Fixed ``SpooledTemporaryFile.seek()`` to return the position instead of
``None``.
```

Grammar fix and slight elaboration on the change made.

Reply:
> slight elaboration on the change made.

While `return` (including implicit return) and `return None` has the same behavior,
"return None intentionally" is different from "return nothing (but None actually)" for human.

In this case, returned None didn't have any meaning.  So I don't want to mention about None.

Reply:
Hmm, good point. I suppose my thoughts were that if I were to read this news entry, I might ask "what did it do previously?". But in this case, the change was probably straightforward enough that they could simply look at the source.